script.js

```diff
@@ -1,4 +1,125 @@
+// Declare Game Modes
+var ROLL_DICE_MODE = 'ROLL_DICE_MODE';
+var SHOW_DICE_MODE = 'SHOW_DICE_MODE';
+var PICK_DICE_ORDER_MODE = 'PICK_DICE_ORDER_MODE';
+// Set Initial Game Mode
+var gameMode = ROLL_DICE_MODE;
+
+// Store Dice Rolls
+var playerDiceRoll = [];
+// Player Re-order
+var playerReorder = [];
+
+// Player Score
+var playerScore = [];
+
+// Player Turn
+var playerTurnSwitch = 1;
+var maxPlayers = 2;
+
+/* ------------------STATEMENTS---------------------------- */
+var statementOne = function (playerTurn, diceOne, diceTwo) {
+  var statementOneOuputValue = `Welcome player ${playerTurn}` + '<br>' + `You rolled Dice 1: ${diceOne} & Dice 2: ${diceTwo}.` + '<br>' + 'Choose the order of the dice';
+  return statementOneOuputValue;
+};
+
+var statmentTwo = function (playerTurn, chosenDice, beatItNum) {
+  if (playerTurn < maxPlayers) {
+    var statementTwo = `Welcome player ${playerTurn}, you chose dice ${chosenDice} first.` + '<br>' + `Your number is ${beatItNum}` + '<br>' + 'It is now player 2\'s turn';
+  } else {
+    var statementTwo = `Welcome player ${playerTurn}, you chose dice ${chosenDice} first.` + '<br>' + `Your number is ${beatItNum}`;
+  }
+  return statementTwo;
+};
+
+var statementThree = function () {
```

Comment:
Best to name these statements more precisely. Not clear from name difference between statements 1, 2, and 3